src/test/java/com/wikia/webdriver/common/core/api/ArticleContent.java

```diff
@@ -70,6 +71,13 @@ public void push(String content) {
     push(content, TestContext.getCurrentMethodName());
   }
 
+  public String createUniqueArticle() {
+    String uniqueText = String.format("%s%s", PageContent.ARTICLE_NAME_PREFIX, new Date().getTime());
+    push(uniqueText, uniqueText);
```

Comment:
I'm aware of Ludwik's plans to refactor the whole thing, so I'm leaving this as is for now.